### ISSUE TITLE
Fable Testing: Page loads at the bottom

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#5182]
## Description
"When I arrived on pages one and two, they both loaded at the bottom of the page; so I had to scroll all the way back up to the top to begin filling out the form. I'm not sure why this happened, but I wanted to bring it to your attention."

"After I typed in my ingredients, I went to the next page, which again loaded at the very bottom of the page. So I used the command, "scroll to top," but when I did that, it refreshed the form. This took me back to the very beginning of the form and I had to literally start over from scratch because none of my answers saved. I know this does not have to do with accessibility or your website and it was just a mistake caused by the way pages are refreshed on mobile, but it would be really helpful if the information auto saved as you go. This would help prevent against dealing with this frustration from having to start over from the beginning."

## Steps to reproduce
1. Go to https://forms-staging.cdssandbox.xyz/en/id/cm7ndt2zt0000lb0bdhwwjuko

## Details
- **Browser**: Chrome
- **Operating system**: iPhone
- **Environment**: Staging
- **Language**: English
- **Viewport**: Mobile
- **Output**: End-User form
- **AT**: Voice Control

## Expected behaviour
Page should load at the top
